godtools/App/Share/SwiftUI Views/TextWithLinks/TextWithLinks.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  TextWithLinks.swift
+//  godtools
+//
+//  Created by Levi Eggert on 6/10/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+struct TextWithLinks: UIViewRepresentable {
```

Comment:
Too bad that SwiftUI can't do this yet!

Reply:
I know I wish there was something provided in SwiftUI.  In iOS 15 url links are supported automatically from Text elements, however, I don't think you can detect the URL tap itself to perform any custom logic.  It just automatically links out to Safari.